### PR TITLE
rename cards and fx hypen

### PR DESCRIPTION
**Pull request by Aiyuuu033119**

### CODE REVIEW

File: airbnb-for-work.html [hunk 1]
```diff
@@ -1020,7 +1020,7 @@ <h3 class="footer__heading">Airbnb</h3>
                           </section>
                         </div>
                         <div class="footer__copyright">
-                          <span class="a8jt5op dir dir-ltr"
+                          <span class="text__highlight dir dir-ltr"
```
Comment (gbertl) [inline airbnb-for-work.html:1023]:
```suggestion
                          <span class="text-highlight dir dir-ltr"
```

no need mag double underscore, wala naman syang parent block. page utilities na ganyan pwede ka magbase sa tailwind or bootstrap sa naming

File: explore.html [hunk 2]
```diff
@@ -82582,7 +82582,7 @@ <h2 class="_y0ndqr">
                                                         >
                                                           <a
                                                             aria-labelledby="title_34444025"
-                                                            class="ln2bl2p dir dir-ltr"
+                                                            class="card__location-click dir dir-ltr"
```
Comment (gbertl) [inline explore.html:82585]: pacheck nalang din kung specific ba sya sa block nayan. kase global na yung card__*

**Review by gbertl — CHANGES_REQUESTED** [review]

File: explore.html (hunk 2 above)
Comment (gbertl) [reply explore.html:82585]: i mean kasi parang pwede syang card-alt-1__location-click

File: airbnb-for-work.html (hunk 1 above)
Comment (Aiyuuu033119) [reply airbnb-for-work.html:1023]: noted po

File: explore.html (hunk 2 above)
Comment (Aiyuuu033119) [reply explore.html:82585]: ginawa ko nalang sya click_card since globally yung ibang cards is clickable

**Pull request closed without merging**